src/EditorFeatures/Test/SolutionCrawler/WorkCoordinatorTests.cs

```diff
@@ -826,9 +826,9 @@ private void TouchEverything(Solution solution)
             {
                 foreach (var document in project.Documents)
                 {
-                    document.GetTextAsync().PumpingWait();
-                    document.GetSyntaxRootAsync().PumpingWait();
-                    document.GetSemanticModelAsync().PumpingWait();
+                    document.GetTextAsync().Wait();
+                    document.GetSyntaxRootAsync().Wait();
+                    document.GetSemanticModelAsync().Wait();
```

Comment:
@jasonmalinowski can you verify these are all correct as a simple Wait?  Don't believe they need to be converted over to await calls.